e2e/app/page/workspace-about-page.ts

```diff
@@ -69,4 +69,26 @@ export default class WorkspaceAboutPage extends AuthenticatedPage{
     await modal.waitUntilVisible();
     return modal;
   }
+
+  async getCdrVersion(): Promise<string> {
+    const cdrVerXpath = '//*[@data-test-id="cdrVersion"]';
+    const cdr = await this.page.waitForXPath(cdrVerXpath, {visible: true});
+    const cdrValue = await (await cdr.getProperty('innerText')).jsonValue();
```

Comment:
why double await ?

Reply:
both are async calls. the inner call has to finish first before outer call starts. I could write it differently with `.then()` (promises chaining) if you think that's a better to read.

Reply:
OK

No need to change